src/FunctionalExtras/Data/Maybe.cs

```diff
@@ -182,6 +195,17 @@ internal struct Just<A> : IMaybe<A>
     public bool Equals(IMaybe<A> other) => other is Just<A>
       && _value.Equals(((Just<A>) other)._value);
 
+    /// <summary>
+    /// Tests the underlying value against the {@code predicate}, returning the {@code Just} of the value for
```

Comment:
C# doesn't use Java's `{@code ... }` style.  These should be `<code>predicate</code>` and the like.

Reply:
Fixed.